api-service/src/services/IngestorService.ts

```diff
@@ -26,18 +22,25 @@ export class IngestorService {
                 console.log("error while connecting to kafka", error.message)
             })
     }
+
+    private handleError(req: Request, res: Response, next: NextFunction, error: any) {
```

Comment:
Shall we move this method to utils? i can see it in the multiple places.

Reply:
Updated accordingly